sdk/eventhubs/azure-messaging-eventhubs/src/samples/java/com/azure/messaging/eventhubs/EventHubConsumerAsyncClientJavaDocCodeSamples.java

```diff
@@ -18,6 +18,10 @@
  * Code snippets demonstrating various {@link EventHubConsumerAsyncClient} scenarios.
  */
 public class EventHubConsumerAsyncClientJavaDocCodeSamples {
+    private final EventHubConsumerAsyncClient consumer = new EventHubClientBuilder()
+        .connectionString("fake-string")
+        .consumerGroup(EventHubClientBuilder.DEFAULT_CONSUMER_GROUP_NAME)
```

Comment:
We should just have  `"consumer-group"` here since the service team doesn't want to promote the use of `DEFAULT_CONSUMER_GROUP_NAME`.